src/frontend/js/order.js

```diff
@@ -3,6 +3,35 @@ $(function() {
     // Javascript of the order application.
     // **************************************
 
+    if ($('.ui.order-override.modal')) {
+      $('.ui.order-override.modal').modal('show');
```

Comment:
The expression  `$('.ui.order-override.modal')` is used in many places, maybe is better to find the way to create the least amount instances of `$('.ui.order-override.modal')`.